packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -207,6 +217,48 @@ export default (fileDetails?: FileDetail, filters?: FileFilter[]): IContextualMe
         metadata.selectors.getAnnotationNameToAnnotationMap
     );
     const loadBalancerBaseUrl = useSelector(interaction.selectors.getLoadBalancerBaseUrl);
+    const [sendMessageToVole, setOnReceiveFromVole] = useMessageExternalSite(
+        `${VOLE_BASE_URL}/write_storage`
+    );
+
+    const openInVole = React.useCallback(async () => {
```

Comment:
How many files have you seen this succeed with? Do we have an idea of when this starts to slow down or break (if at all?)?

Reply:
I'm wondering if this is overkill for if the user just wants to pass a single file to Vol-E. But maybe if there isn't a slow-down it doesn't matter?